components/idp-mgt/org.wso2.carbon.idp.mgt.ui/pom.xml

```diff
@@ -125,15 +125,12 @@
             <plugin>
```

Comment:
Since the parent pom of the repo has the plugin enabled, let's remove the config from here.

Reply:
Removed the plugin